piker/brokers/kraken.py

```diff
@@ -119,6 +119,19 @@ class Pair(BaseModel):
     ordermin: float  # minimum order volume for pair
 
 
+class Order(BaseModel):
```

Comment:
I would probably consider trying to type-validate these where possible though it's not a huge deal yet.

Reply:
I think I do this somewhere

Reply:
`pydantic` will do it faster in C extensions too.